compiler/passes/codegen.cpp

```diff
@@ -607,11 +602,13 @@ static void codegen_header_compilation_config() {
     fprintf(cfgfile.fptr,
             "printf(\"%%s\", \"  CHPL_HOME: %s\\n\");\n",
             CHPL_HOME);
-    for (int i = 0; i < num_chpl_env_vars; i++) {
-      fprintf(cfgfile.fptr,
-              "printf(\"%%s\", \"  %s: %s\\n\");\n",
-              chpl_env_var_names[i],
-              chpl_env_vars[i]);
+    for (std::map<std::string, const char*>::iterator env=envMap.begin(); env!=envMap.end(); ++env) {
+      if (strcmp((*env).first.c_str(), "CHPL_HOME") != 0) {
```

Comment:
Is there any reason to not print out CHPL_HOME here, and remove the few lines above that explicitly print it out?